python/ray/data/_internal/stats.py

```diff
@@ -93,36 +93,36 @@ class _DatasetStatsBuilder:
 
     def __init__(
         self,
-        stage_name: str,
+        operator_name: str,
         parent: "DatasetStats",
         override_start_time: Optional[float],
     ):
-        self.stage_name = stage_name
+        self.operator_name = operator_name
         self.parent = parent
         self.start_time = override_start_time or time.perf_counter()
 
-    def build_multistage(self, stages: StatsDict) -> "DatasetStats":
-        stage_infos = {}
-        for i, (k, v) in enumerate(stages.items()):
+    def build_multioperator(self, metadata: StatsDict) -> "DatasetStats":
+        op_metadata = {}
+        for i, (k, v) in enumerate(metadata.items()):
             capped_k = capfirst(k)
-            if len(stages) > 1:
+            if len(metadata) > 1:
```

Comment:
do we have any case where `metadata` has more than 1 key-value pair? if not, we could potentially simplify this down.

Reply:
Yeah, you showed me this [example](https://github.com/ray-project/ray/blob/master/python/ray/data/_internal/planner/exchange/pull_based_shuffle_task_scheduler.py#L104) earlier

Reply:
ah right, then this looks good to me. thanks